README.md

```diff
@@ -77,6 +77,8 @@ In the Loop project's Build Settings, change the value of `MAIN_APP_BUNDLE_IDENT
 
 Loop optionally supports select third-party remote services. While none of them are required to run the app, including [mLab](https://mlab.com) keys is strongly recommended at this time so loop diagnostic data can be stored in case retrospective analysis is needed.
 
+If the Nightscout related entries are included, treatments and other pump data will be uploaded to your Nightscout site.
```

Comment:
Note that this is only true if "Insulin Data Source" is set to "History" in Settings.